lib/batch_verification/src/sequencer/component.rs

```diff
@@ -201,13 +220,34 @@ impl BatchVerifier {
                 tracing::info!("BatchForSigning channel closed, exiting verifier",);
                 break Ok(());
             };
+
+            // We skip signing batches that were already committed. This happens on startup
+            if batch_envelope.batch_number() <= self.last_committed_batch_number {
+                tracing::info!(
+                    "Skipping signing of already committed batch {}",
+                    batch_envelope.batch_number()
+                );
+                singed_batcher_sender
+                    .send(
+                        batch_envelope
+                            .with_signatures(BatchSignatureData::NotNeeded)
```

Comment:
Hmmm not sure it's right to use `NotNeeded` here. The comment there says that it's for backwards compatibility - but now we use it for something else. I guess a quick fix is to add one more enum variant